src/main/java/frc/robot/subsystems/DiffDriver.java

```diff
@@ -0,0 +1,92 @@
+
+    public DiffDriver() {
+        Logger.setup("Constructing Subsystem: DiffDriver...");
+
+        // Determine whether or not to disable the subsystem
+        m_disabled = (SubsystemDevices.diffDrive == null);
+        if (m_disabled) {
+            Logger.error("DiffDriver devices not initialized! Disabling subsystem...");
+            return;
+        }
+
+        // Configure the subsystem devices
+        // TODO: Investigate why these motor controllers have to be inverted.
+        //       Are all TalonFx Motor Controllers backwards?
+        SubsystemDevices.talonFxDiffWheelFrontLeft.setInverted(true);
+        SubsystemDevices.talonFxDiffWheelFrontRight.setInverted(true);
+        SubsystemDevices.talonFxDiffWheelRearLeft.setInverted(true);
+        SubsystemDevices.talonFxDiffWheelRearRight.setInverted(true);
+        SubsystemDevices.talonFxDiffWheelRearLeft.follow(SubsystemDevices.talonFxDiffWheelFrontLeft);
+        SubsystemDevices.talonFxDiffWheelRearRight.follow(SubsystemDevices.talonFxDiffWheelFrontRight);
+
+        SubsystemDevices.talonFxDiffWheelFrontLeft.configOpenloopRamp(SECONDS_FROM_NEUTRAL_TO_FULL, TIMEOUT_MS);
+        SubsystemDevices.talonFxDiffWheelRearLeft.configOpenloopRamp(SECONDS_FROM_NEUTRAL_TO_FULL, TIMEOUT_MS);
+        SubsystemDevices.talonFxDiffWheelFrontRight.configOpenloopRamp(SECONDS_FROM_NEUTRAL_TO_FULL, TIMEOUT_MS);
+        SubsystemDevices.talonFxDiffWheelRearRight.configOpenloopRamp(SECONDS_FROM_NEUTRAL_TO_FULL, TIMEOUT_MS);
+    }
+
+    @Override
+    public void periodic() {
+        // This method will be called once per scheduler run
+    }
+
+    // Flip the control direction of the joystick in Y (or Y Left for Xbox thumbsticks)
+    public Boolean flipControlStickDirection() {
+        Logger.action("Toggling DiffDriver control stick direction...");
+
+        controlStickDirectionFlipped = !controlStickDirectionFlipped;
+
+        if (controlStickDirectionFlipped) {
+            Logger.info("DiffDriver control stick direction is now flipped.");
+        } else {
+            Logger.info("DiffDriver control stick direction is now standard (not flipped).");
+        }
+
+        return controlStickDirectionFlipped;
+    }
+
+    // Stop all the drive motors
+    public void stop() {
+        if (m_disabled) {
+            // TODO: Not only should this not be necessary, but it should also produce
+            //       a runtime error, because diffDrive should be null if the system is disabled.
+            SubsystemDevices.diffDrive.feed();
+            return;
+        }
+
+        SubsystemDevices.diffDrive.stopMotor();
+    }
+
+    // Drive using the tank method
+    public void driveTank(double leftSpeed, double rightSpeed) {
+        if (m_disabled) {
+            // TODO: Not only should this not be necessary, but it should also produce
+            //       a runtime error, because diffDrive should be null if the system is disabled.
+            SubsystemDevices.diffDrive.feed();
```

Comment:
Not only should this not be necessary, but it should also produce a runtime error, because diffDrive should be null if the system is disabled.